main.cxx

```diff
@@ -46,6 +46,9 @@ static inline bool handle_raw_input_2(RAWINPUT const& raw)
 		       (long int)raw.data.mouse.lLastX,
 		       (long int)raw.data.mouse.lLastY,
 		       (unsigned long int)raw.data.mouse.ulExtraInformation);
+		if (raw.data.mouse.usButtonFlags == RI_MOUSE_WHEEL)
```

Comment:
You either forgot to add curly brackets to the previous if or indented this wrong.

Reply:
I had them and then deleted them while I was moving things around and forgot to put them back. Bleh.